public/app/features/templating/templateSrv.js

```diff
@@ -107,10 +111,15 @@ function (angular, _) {
       });
     };
 
-    this.getAllValue = function(variable) {
+    this.getAllValue = function(variable, format) {
       if (variable.allValue) {
         return variable.allValue;
       }
+
+			if (format === 'regex') {
+				return '.*';
```

Comment:
this is not right, if your variable includes for example 4 servers out of 1000, then All should not be `.*` which should match all 1000 servers.